src/cache.c

```diff
@@ -20,7 +25,13 @@
 /* THRESHOLD is set to identify hot spots. Once the frequency of use for a block
  * exceeds the THRESHOLD, the JIT compiler flow is triggered.
  */
-#define THRESHOLD 1000
+#define THRESHOLD 10000
```

Comment:
The change of `THRESHOLD` should come with an engineering consideration.

Reply:
Perhaps we keep the value at 1000 for now and make modifications when we optimize the JIT.